Tests/XMLCoderTests/AdvancedFeatures/SimpleChoiceTests.swift

```diff
@@ -85,4 +85,23 @@ final class SimpleChoiceTests: XCTestCase {
         let decoded = try XMLDecoder().decode([[IntOrString]].self, from: encoded)
         XCTAssertEqual(original, decoded)
     }
+    
+    func testMixedChoice() throws {
+        enum Choice: Equatable, Codable {
```

Comment:
I think that the issue is that Xcode 11.x can't yet generate the codable implementation for this type.
We can wrap this test in a compiler directive like `#if compiler` or `#if swift`.

Reply:
done